source/Government.cpp

```diff
@@ -72,8 +72,6 @@ namespace {
 				penalty += it.second;
 		return penalty;
 	}
-
-	unsigned nextID = 0;
```

Comment:
casually changing the way governments have always worked in a PR with a new feature, are we?
~~I can almost garantee this will break smth~~ if you really want to go this way smbd should test it extensively, I think

Reply:
yeah I'm aware, of course it needs more testing

Reply:
I think it'd be a good idea to justify why this change was made in the description

I havent thought too much about it but I'd just have used a DEFAULT_RELATION constant per government

Reply:
nvm I think I found a way to do this with vectors, although it's quite hacky ~~like the whole thing with IDs~~.
I'd just close this for now and open the refactor separately.

Reply:
Sounds less risky that way, but you could have just updated this PR yk since the functionality is the same

Reply:
I've tested the refactor separately, and... surprisingly everything works, but then I found this: https://github.com/endless-sky/endless-sky/commit/49dc75cdb94972c45a49dd403277ea9fb292a424, seems like it was done intentionally.
What do you think?

Reply:
Yeah it was intentionaly done this way

Reply:
So, do we want to revert https://github.com/endless-sky/endless-sky/commit/49dc75cdb94972c45a49dd403277ea9fb292a424 or continue using IDs?

Reply:
I prefer using IDs so its all about if you can justify it